extensions/exquery/restxq/src/main/java/org/exist/extensions/exquery/restxq/impl/RestXqServiceImpl.java

```diff
@@ -164,9 +171,24 @@ protected Sequence extractRequestBody(final HttpRequest request) throws RestXqSe
                     if (contentType.contains(";")) {
                         contentType = contentType.substring(0, contentType.indexOf(";"));
                     }
-
                     MimeType mimeType = MimeTable.getInstance().getContentType(contentType);
-                    if (mimeType != null && !mimeType.isXMLType()) {
+
+                    if(contentType.equalsIgnoreCase("multipart/related")){
```

Comment:
“Related” ????? Never seen that before?

Reply:
Thank you for the quick response.
The MIME multipart/related is needed for SOAP-MTOM/XOP messages
[SOAP Message Transmission Optimization Mechanism](https://www.w3.org/TR/soap12-mtom/#xop-serialization)